packages/relay/src/lib/clients/cache/localLRUCache.ts

```diff
@@ -232,9 +237,10 @@ export class LocalLRUCache implements ICacheClient {
    * @param {RequestDetails} requestDetails - The request details for logging and tracking.
    */
   public async delete(key: string, callingMethod: string, requestDetails: RequestDetails): Promise<void> {
-    this.logger.trace(`${requestDetails.formattedRequestId} delete cache for ${key} on ${callingMethod} call`);
-    const cache = this.getCacheInstance(key);
```

Comment:
Same here please kindly revert the changes. I recommend leaving the unrelated code unchanged to avoid potential bugs.